src/plugins/firefoxbookmarks/src/main.cpp

```diff
@@ -273,7 +272,7 @@ FirefoxBookmarks::Extension::Extension()
         QStringList ids = profilesIni.childGroups();
         if ( ids.isEmpty() )
             qWarning() << qPrintable(QString("[%1] No Firefox profiles found.").arg(Core::Extension::id));
-        else {
+        else { attach parent names
```

Comment:
I have no idea what this does here... I have wrote this commit with the github web editor... maybe i dragged that over there...